### PR TITLE
Drop the py26 and py27 in tox.ini file

### PR DESCRIPTION
**Pull request by LinglingXiang**
1.Drop the support of python 2.6 and python 2.7

### CODE REVIEW

**Comment by LinglingXiang** [comment]
@rbikar , please help review, thanks

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,7 @@ dist: trusty
 language: python
 matrix:
   include:
-    - python: "2.6"
-      env: TOXENV=py26
-    - python: "2.7"
-      env: TOXENV=py27
-    - python: "3.6"
+   - python: "3.6"
```
Comment (rbikar) [inline .travis.yml:5]: it looks like this has wrong indentation

**Review by rbikar — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -2,10 +2,6 @@ dist: trusty
 language: python
 matrix:
   include:
-    - python: "2.6"
-      env: TOXENV=py26
-    - python: "2.7"
-      env: TOXENV=py27
     - python: "3.6"
       env: TOXENV=py36
     - python: "2.7"
```
Comment (rbikar) [inline .travis.yml:7]: I guess you could use python3 here as well.

**Review by rbikar — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+Please raise issues via the [new interface](./../issues/new/choose)
```
Comment (rbikar) [inline .github/ISSUE_TEMPLATE.md:2]: This file is not needed, please don't add it.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (rbikar) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: This file is not needed, please don't add it.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (rbikar) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: This file not needed, please don't add it.

File: .github/ISSUE_TEMPLATE/question_support.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (rbikar) [inline .github/ISSUE_TEMPLATE/question_support.md:1]: This is not needed, please don't add it.

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -0,0 +1,27 @@
+name: Release on PyPI
```
Comment (rbikar) [inline .github/workflows/release.yml:1]: This is not needed, we have never created a package of dockpulp on pypi.

File: .github/workflows/tox-test.yml [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+        run: tox -e cov
+      # Codcov Action required installing pytest-cov as it needs coverage.
+      # The coverage library installed within tox is in virt-env and not
+      # accessible to Codecov.
+      - name: Install pytest cov
+        run: pip install pytest-cov
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```
Comment (rbikar) [inline .github/workflows/tox-test.yml:74]: token shouldn't be needed for public repos

File: .github/workflows/tox-test.yml [hunk 9]
```diff
@@ -0,0 +1,88 @@
+name: Tox tests
```
Comment (rbikar) [inline .github/workflows/tox-test.yml:1]: if you want to add actions with:
- ` tox -e static` or `tox -e pidiff` (which is likely not needed) or `tox -e docs` you need to define them in tox.ini
 
But I think we only need actions with:
-  `tox -e py38`
-  `tox -e py39`
-  `tox -e flake8`

File: test-requirements.txt [hunk 10]
```diff
@@ -0,0 +1,8 @@
+pytest==3.2.5
```
Comment (rbikar) [inline test-requirements.txt:1]: what actually uses this file?

**Review by rbikar — CHANGES_REQUESTED** [review]

File: .github/workflows/tox-test.yml [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install Tox
+        run: pip install tox
+      - name: Run Tox
+        run: tox -e cov
```
Comment (rbikar) [inline .github/workflows/tox-test.yml:53]: this env is not define in tox.ini.
Either remove this `coverage`  check or add env definition to tox.ini.

File: test-requirements.txt (hunk 10 above)
Comment (rbikar) [reply test-requirements.txt:1]: Is this file really required?

**Review by rbikar — CHANGES_REQUESTED** [review]

**Comment by LinglingXiang** [comment]
@rbikar all the errors fixed, please help review again

**Review by rbikar — CHANGES_REQUESTED** [review]

**Comment by rbikar** [comment]
@LinglingXiang 
The related PR was merged, please rebase this PR on current master and also address the issue where I added comment.

**Pull request closed without merging**